Remotion/Core/Collections.Caching/ConcurrentCache.cs

```diff
@@ -147,7 +149,7 @@ public ConcurrentCache ([NotNull] IEqualityComparer<TKey> comparer)
     /// <returns>
     /// true if an element with the specified key was found; otherwise, false.
     /// </returns>
-    public bool TryGetValue (TKey key, out TValue value)
+    public bool TryGetValue (TKey key, [AllowNull, MaybeNullWhen(false)] out TValue value)

```

Comment:
order of allownull, maybenullwhen should be consistent. probably reads better with allownull and then maybenull. check other files. also whitespace before parens open.